src/components/locations-list/component.js

```diff
@@ -0,0 +1,57 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Link from 'redux-first-router-link';
+import classnames from 'classnames';
+import styles from './style.module.scss';
+
+class LocationsList extends PureComponent {
```

Comment:
I believe if there is no state (PureComponent) using function components is a way more readable approach. As in my other comment a matter of personal preference.